packages/desktop/components/popups/send/SendConfirmationPopup2.svelte

```diff
@@ -0,0 +1,235 @@
+<script lang="ts">
+    import { get } from 'svelte/store'
+    import {
+        Button,
+        ExpirationTimePicker,
+        KeyValueBox,
+        Text,
+        TextHint,
+        FontWeight,
+        TextType,
+        NftActivityDetails,
+        BasicActivityDetails,
+        ActivityInformation,
+        Toggle,
+    } from 'shared/components'
+    import { Tab, TextHintVariant } from 'shared/components/enums'
+    import { prepareOutput, selectedAccount } from '@core/account'
+    import { localize } from '@core/i18n'
+    import { checkActiveProfileAuth, isActiveLedgerProfile } from '@core/profile'
+    import { TimePeriod } from '@core/utils'
+    import { TokenStandard } from '@core/wallet/enums'
+    import { newTransactionDetails, NewTransactionType, updateNewTransactionDetails } from '@core/wallet/stores'
+    import { sendOutput } from '@core/wallet/actions'
+    import {
+        validateSendConfirmation,
+        getStorageDepositFromOutput,
+        getOutputParameters,
+        getDefaultTransactionOptions,
+    } from '@core/wallet/utils'
+    import { closePopup, openPopup, PopupId, updatePopupProps } from '@auxiliary/popup'
+    import { NewTokenTransactionDetails, NftActivity, TransactionActivity, VestingActivity } from '@core/wallet/types'
+    import { CommonOutput, Output } from '@iota/sdk/out/types'
+    import { ledgerPreparedOutput } from '@core/ledger'
+    import { handleError } from '@core/error/handlers/handleError'
+    import { ToggleColor } from '@ui/inputs/Toggle.svelte'
+    import { getInitialExpirationDate, rebuildActivity } from './sendFormUtils'
+
+    export let disableBack = false
+    export let isSendAndClosePopup: boolean = false
+    export let preparedOutput: Output
+    export let calculatedStorageDeposit: number = 0
+
+    let activeTab: Tab
+    let activity: TransactionActivity | VestingActivity | NftActivity | undefined = undefined
+    let expirationTimePicker: ExpirationTimePicker
+    let initialExpirationDate: TimePeriod
+
+    const transactionDetails = get(newTransactionDetails)
+    const {
+        type: transactionType,
+        recipient,
+        layer2Parameters,
+        disableToggleGift,
+        disableChangeExpiration,
+    } = get(newTransactionDetails)
```

Comment:
I would put the variables in order. First the `const`, and then `let` them together